.gitignore

```diff
@@ -58,4 +58,7 @@ MANIFEST
 
 # Outputs
 *.csv
-!tests/fixtures/*.csv
\ No newline at end of file
+!tests/fixtures/*.csv
+
```

Comment:
Did you generate the .gitignore file automatically?

Reply:
Yes, using the pyscaffold package